pkg/vacuum/vacuum.go

```diff
@@ -0,0 +1,223 @@
+package vacuum
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/timescale/promscale/pkg/log"
+	"github.com/timescale/promscale/pkg/pgxconn"
+)
+
+const (
+	sqlStart            = "SELECT pg_try_advisory_lock(1982010619820106)"
```

Comment:
I wonder if it's worth defining the id in the extension code like `_prom_catalog.get_advisory_lock_prefix_maintenance` or at least using a similar prefix (12379?)